{"uid":"dc0fdb8492d56c56","name":"should display the application name","fullName":"My First Test.should display the application name","historyId":"472e2eea17848fe79ae69a098d829169","time":{"start":1743423303270,"stop":1743423311741,"duration":8471},"description":"","descriptionHtml":"","status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":2,"retriesStatusChange":true,"beforeStages":[],"afterStages":[],"labels":[{"name":"suite","value":"My First Test"},{"name":"testClass","value":"My First Test"},{"name":"testMethod","value":"should display the application name"},{"name":"package","value":"My First Test"},{"name":"resultFormat","value":"allure1"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"2f1f76019358309c","status":"passed","time":{"start":1743422707352,"stop":1743422714784,"duration":7432}},{"uid":"2409f374cf328fd1","status":"broken","statusDetails":"`cy.visit()` failed trying to load:\n                \n                https://getlinked-main-dev-frontend.vercel.app/\n                \n                We attempted to make an http request to this URL but the request failed without a response.\n                \n                We received this error at the network level:\n                \n                  > Error: getaddrinfo EAI_AGAIN getlinked-main-dev-frontend.vercel.app\n                \n                Common situations why this would fail:\n                  - you don't have internet access\n                  - you forgot to run / boot your web server\n                  - your web server isn't accessible\n                  - you have weird network configuration settings on your computer\n                \n                Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `My First Test`","time":{"start":1743422259890,"stop":1743422261082,"duration":1192}}],"categories":[],"tags":[]},"source":"dc0fdb8492d56c56.json","parameterValues":[]}